cloud/translation/gnmt/tensorflow/README.md

```diff
@@ -16,7 +16,7 @@ CPU
 pip install --user tensorflow
 ```
 
-### Run GNMT over full Dataset
+### Run GNMT over full Dataset (outdated)
```

Comment:
run_task.py is still up-to-date. However, I haven't tried fetching the model yet. That is actually needed for loadgen_gnmt.py as well. So I think we'll need to verify whether fetching the model still works.

Reply:
ok